packages/corvid-cli/src/apps/clone.js

```diff
@@ -75,10 +76,17 @@ async function extractMetasiteIdAndName(url, cookie) {
 async function clone(spinner, args, cookie) {
   spinner.start(chalk.grey(getMessage("Clone_Getting_Site_Data")));
   try {
-    const { metasiteId } = await extractMetasiteIdAndName(args.url, cookie);
+    const { metasiteId, siteName } = await extractMetasiteIdAndName(
+      args.url,
+      cookie
+    );
+
+    if (siteName == null) {
+      throw new UserError(getMessage("Clone_Bad_URL_Error", { url: args.url }));
```

Comment:
Bad URL sounds like something that we should show when the given string is not a URL

Reply:
sure, we need something better. It can be really bad url, non-wix url or just someones else's url